### ISSUE TITLE
Probable false positive/misleading description of detected SCA mitigations

### ISSUE THREAD

**Issue by impulse200** [HewlettPackard/LinuxKI#58]
Hi.
There is a message in kp.html that SCA mitigation is active on the host I currently diagnosing:

```
Side-Channel Attack (Spectre/Meltdown) Mitigations:
/sys/devices/system/cpu/vulnerabilities/gather_data_sampling Vulnerable
/sys/devices/system/cpu/vulnerabilities/itlb_multihit KVM: Mitigation: VMX disabled
/sys/devices/system/cpu/vulnerabilities/l1tf Mitigation: PTE Inversion; VMX: vulnerable
/sys/devices/system/cpu/vulnerabilities/mds Vulnerable; SMT vulnerable
/sys/devices/system/cpu/vulnerabilities/meltdown Vulnerable
/sys/devices/system/cpu/vulnerabilities/mmio_stale_data Vulnerable
/sys/devices/system/cpu/vulnerabilities/retbleed Vulnerable
/sys/devices/system/cpu/vulnerabilities/spec_rstack_overflow Not affected
/sys/devices/system/cpu/vulnerabilities/spec_store_bypass Vulnerable
/sys/devices/system/cpu/vulnerabilities/spectre_v1 Vulnerable: __user pointer sanitization and usercopy barriers only; no swapgs barriers
/sys/devices/system/cpu/vulnerabilities/spectre_v2 Vulnerable, IBPB: disabled, STIBP: disabled, PBRSB-eIBRS: Not affected
/sys/devices/system/cpu/vulnerabilities/srbds Not affected
/sys/devices/system/cpu/vulnerabilities/tsx_async_abort Vulnerable
Warning: Side Channel Attack (Spectre/Meltdown) mitigations present [[INFO]](http://htmlpreview.github.io/?https://github.com/HewlettPackard/LinuxKI/blob/master/documentation/sca_vuln.htm)[[Next]](http://192.168.1.60:8000/kp.0930_1328.html#warncheck1)
```

But report sample above shows spectre_v1 and spectre_v2 as "Vulnerable" without "Mitigation:" clause. And that system is booted with mitigation=off kernel parameter. 

So the question is: are SCA mitigations actually enabled? The CPI column in "Global CPU Usage by CPU" report is around 3.09 - 3.93 if that matters.

**Comment by MarkCRay** [comment]
Thanks for the report.   The algorithm is pretty basics, and looks at all the vulnerabilities and print the Warning it the word "Mitigation" is present, which it is in this case:

/sys/devices/system/cpu/vulnerabilities/itlb_multihit KVM: Mitigation: VMX disabled
/sys/devices/system/cpu/vulnerabilities/l1tf Mitigation: PTE Inversion; VMX: vulnerable

I need to change the references to "Side Channel Attack (Spectre/Meltdown)" to simply says "Security" Mitigations.

**Comment by impulse200** [comment]
Actually https://docs.kernel.org/admin-guide/hw-vuln/multihit.html#itlb-multihit-system-information says that "KVM: Mitigation: VMX disabled" means that "KVM is not vulnerable because Virtual Machine Extensions (VMX) is disabled."
Linux implementation of mitigations is a mess.

**Comment by MarkCRay** [comment]
Fixed in version 7.8, released on Nov 9, 2023.
